pkg/ui/chooser/embed/index.html

```diff
@@ -76,18 +76,25 @@
                 <p>This is the modern Aim UI, much faster than MLFlow.</p>
             </article>
         </div>
-        <p>Selected namespace: <b><u>{{.CurrentNamespace.Description}}</u></b></p>
+        <p>Selected namespace: 
+            <b>
+                <u>{{.CurrentNamespace.Code}}</u>
+                {{ if ne "" .CurrentNamespace.Description }}
+                    - {{ .CurrentNamespace.Description }}
+                {{ end }}
```

Comment:
I think we should apply this logic everywhere for consistency, probably by computing this before passing it to the template (as e.g. `DisplayName`)

Reply:
afaik this is the only place -- it looked strange to do this in the listing because the lengths were so different

Reply:
But the listing is probably where the description is the most useful if we display it, no?
I think we either don't display the description in the chooser at all, or we display it _everywhere_ in the chooser.

Reply:
yes left-align will probably look better, i'll try it

Reply:
![image](https://github.com/G-Research/fasttrackml/assets/1463452/4faaa2a0-d332-4d94-9314-8316bf7dc779)
![image](https://github.com/G-Research/fasttrackml/assets/1463452/7bb18a41-d307-4938-8175-69bacd2bd53f)

Reply:
could you please make the _text_ left-aligned and move the arrow to the left of it?
also, I _think_ it might look better with the description in parentheses, and with the default namespace being described as "Default namespace".

e.g.
```
  custom 1
  custom2 (has description)
→ default (Default namespace)
```

Reply:
how's this?
![image](https://github.com/G-Research/fasttrackml/assets/1463452/c267d568-27fe-40fb-bdfe-3090c6c4d83f)